docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/install-on-slurm.rst

```diff
@@ -155,7 +156,15 @@ fulfilled and configured, install and configure the Determined master:
    see :ref:`slurm-image-config`.
 
 #. Log into Determined, see :ref:`users`. The Determined user must be linked to a user on the HPC
-   cluster, see :ref:`run-as-user`.
+   cluster. If logged with a Determined admininstrator account, the following example creates a
+   Determined user account properly linked for the current user. For more details see
```

Comment:
I think this would read better if it said, "... user account that is linked to the current user's Linux account".